core/embed/rust/src/ui/layout/obj.rs

```diff
@@ -291,6 +303,7 @@ impl LayoutObj {
                 Qstr::MP_QSTR_request_complete_repaint => obj_fn_1!(ui_layout_request_complete_repaint).as_obj(),
                 Qstr::MP_QSTR_trace => obj_fn_2!(ui_layout_trace).as_obj(),
                 Qstr::MP_QSTR_bounds => obj_fn_1!(ui_layout_bounds).as_obj(),
+                Qstr::MP_QSTR_delete => obj_fn_1!(ui_layout_delete).as_obj(),
```

Comment:
call it `MP_QSTR___del__`

Reply:
fixed https://github.com/trezor/trezor-firmware/commit/ee7f88dddc0c6a9e9708e8a30c045a4e411ce7fe